viadot/task_utils.py

```diff
@@ -792,3 +792,28 @@ def validate_df(df: pd.DataFrame, tests: dict = None) -> None:
         raise ValidationError(
             f"Validation failed for {failed_tests} test/tests: {failed_tests_msg}"
         )
+
+
+def check_value(base, lvls: List):
```

Comment:
* Please add return type/types.
* Add type hint to base 
* Maybe change name to `levels`